### PR TITLE
Install.rst: Add link to bash/zsh auto-completion

### PR DESCRIPTION
**Pull request by saurabhraj1896**
This link explains users about setting-up
tab-completion for coala arguements and bear names.

Closes https://github.com/coala/documentation/issues/386

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: Users/Install.rst [hunk 1]
```diff
@@ -285,6 +285,13 @@ For x86 python or for x64 python:
 
 Replace "XXX" with the ClangSharp version you received from nuget.
 
+Shell-autocompletion support
+----------------------------
+
+If you are a bash/zsh user , checkout the
+`guide <http://docs.coala.io/en/latest/Users/Shell_Autocompletion.html>`
+for setting-up tab-completion for coala arguements and bear names.
```
Comment (supergr35) [inline Users/Install.rst]: `tab-completion` -> `autocompletion` (make this change in the commit body and shortlog as well)

Comment (supergr35) [inline Users/Install.rst]: `setting-up`-> `setting up`

Comment (saurabhraj1896) [inline Users/Install.rst]: @supergr35 do i need to squash my last commit with this new commit(tab-completion -> autocompletion) or do separate commit for that ?

Comment (saurabhraj1896) [inline Users/Install.rst]: i mean to say that do i need to do separate commit apart from last commit or just amend my last commit?

**Comment by saurabhraj1896** [comment]
review @supergr35

**Comment by supergr35** [comment]
ack 9011cf9

File: Users/Install.rst [hunk 2]
```diff
@@ -285,6 +285,14 @@ For x86 python or for x64 python:
 
 Replace "XXX" with the ClangSharp version you received from nuget.
 
+Shell-Autocompletion Support
+----------------------------
+
+If you are a bash/zsh user , checkout the
+``guide`` (https://docs.coala.io/en/latest/Users/Shell_Autocompletion.html)
```
Comment (aptrishu) [inline Users/Install.rst]: link the text to the url..

**Comment by aptrishu** [comment]
ack 3dd940117dc8c0c7ba3069537688c6f835e4bfa0

**Comment by aptrishu** [comment]
@rultor merge
